### ISSUE TITLE
updateNewMessage returns message.id which is not qnique for the chat it belongs

### ISSUE THREAD

**Issue by duataden** [tdlib/td#2907]
In tdlib docs https://core.telegram.org/tdlib/docs/classtd_1_1td__api_1_1message.html:
for message public fields:

> int53    |      id_       |     Message identifier; unique for the chat to which the message belongs.

However, tdlib json interface (for python) returns some big number as message.id, not related to the actual unique id for the chat (maybe globally unique, no idea). There are no other id fields with the correct id

`{
  "@type": "updateNewMessage",
  "message": {
    "id": 59382956032,
    "date": 1717109728,
    "@type": "message",`

Without this id it is not possible to construct an URL to the message like https://t.me/<chat.username>/<message.id>

**Comment by levlam** [comment]
All message identifiers are unique within the chat for a given TDLib instance. Each message has a unique identifier that must be used to reference the message in the given TDLib instance.

> Without this id it is not possible to construct an URL to the message like https://t.me/<chat.username>/<message.id>

You should never construct t.me URLs yourself. TDLib has `getInternalLink`/`getMessageLink`/`getProxyLink` and similar methods for all types of existing links.

**Comment by duataden** [comment]
Yes, id is probably unique for the chat (as well as in some wider scope of tdlib instance), but I compare it to MTProto where message id is an incremental number for a chat and easy to use in the URL  https://t.me/<chat.username>/<message.id> without extra calls. So, it is <chat.id, message.id> what is globally unique and quite easy to work with.

Anyway, thanks, I'll try so do as you advides
